### DISCUSSION TITLE
CSS - how to select outgoing edge of the active node

### DISCUSSION THREAD

**Discussion by fieldrabbit21** [HEmile/juggl#89] [Q&A] [answered]
Hi all,

I know how to select edges of an active node in CSS.
From these edges, how do I select only the outgoing (outward pointing) edges?
Thanks!

**Comment by HEmile** [comment, marked as answer]
That's a good question, I don't think this is currently possible with CSS. What do you want to achieve?

**Reply by fieldrabbit21** [reply]
Thanks for your feedback Emile.
What I like to achieve: If I can select outgoing edges of the active node, I could give it a color, I would be able to see the "child" nodes in a glance, and the parent nodes in a glance. For understanding the relationship between nodes.
Regards, Henry
